src/translations/en.json

```diff
@@ -2079,6 +2079,9 @@
         }
       },
       "lovelace": {
+        "badges": {
+          "summary": "{count} {count, plural,\n one {domain}\n other {domains}\n} {state}"
```

Comment:
`domain` and `domains` is the translation here. `count` is the only variable. (I know, the syntax is confusing...) so we would have to add a key for every domain:

```
"light": "{count} {count, plural,\n one {light}\n other {lights}\n} {state}"
"switch": "{count} {count, plural,\n one {switch}\n other {switches}\n} {state}"
```

Reply:
But this should be in the backend, as that is where the domain/state translations are.

Reply:
I'm not sure how to approach that on the backend :shrug:

Reply:
@bramkragten should we shelve this? I'm not clear on how to overcome these translation obstacles.